ocs_ci/deployment/deployment.py

```diff
@@ -599,6 +599,13 @@ def deploy_ocs_via_operator(self, image=None):
                 and not lso_type == constants.AWS_EBS
             ):
                 deviceset_data["count"] = 2
+            # setting resource limits for AWS i3
+            # https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.6/html-single/deploying_openshift_container_storage_using_amazon_web_services/index#creating-openshift-container-storage-cluster-on-amazon-ec2_local-storage
+            if ocs_version >= 4.5 and self.platform.lower() == constants.AWS_PLATFORM:
```

Comment:
Can we add it only for:
worker_instance_type == 'i3en.2xlarge'
?
As we have also arbiter lso deployment on AWS and this should not be there.

Reply:
Done